backend/src/apiserver/server/test_util.go

```diff
@@ -73,6 +73,15 @@ func initWithExperiment(t *testing.T) (*resource.FakeClientManager, *resource.Re
 	return clientManager, resourceManager, experiment
 }
 
+func initWithExperiment_KFAM_Unauthorized(t *testing.T) (*resource.FakeClientManager, *resource.ResourceManager, *model.Experiment) {
+	clientManager := resource.NewFakeClientManagerOrFatal_KFAM_Authorized(util.NewFakeTimeForEpoch())
```

Comment:
this is a bit confusing. Test is for unauthorized, the function is authorized, and func implementation is again unauthzed. 

could you double check if this is what you want?

Reply:
this is some typo. Fixed.